code/modules/vehicles/cars/car.dm

```diff
@@ -93,3 +96,117 @@
 		M.visible_message("<span class='warning'>[M] is forced into \the [src]!</span>")
 	M.forceMove(src)
 	add_occupant(M, VEHICLE_CONTROL_KIDNAPPED)
+
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedastrian focused station, and now all these cars are ruining it!." //ok boomer
+	icon_state = "car"
+	max_integrity = 300
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	enter_delay = 50 //No speed escapes
+	max_occupants = 5
+	movedelay = 1.15
+	material_flags = MATERIAL_AFFECT_STATISTICS | MATERIAL_COLOR | MATERIAL_ADD_PREFIX
+	var/obj/item/card/id/linked_id = null
+	var/can_drive = FALSE
+
+
+/obj/vehicle/sealed/car/civ/Initialize()
+	var/datum/material/M = pick(subtypesof(/datum/material))
+	custom_materials = list()
+	custom_materials[M] = 10000
+	. = ..()
+
+
+/obj/vehicle/sealed/car/civ/generate_actions()
+	. = ..()
+	initialize_controller_action_type(/datum/action/vehicle/sealed/horn, VEHICLE_CONTROL_DRIVE)
+
+/obj/vehicle/sealed/car/civ/Cross(atom/movable/AM)
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE INFRONT OF MY CAR YA FUCKIN KNOB", "GET OUTA DA WAY I'M DRIVIN' HERE", "YOU ARE RUINING MY SUSPENSION CUNT!"), forced="car crash")
+		L.adjustBruteLoss(3) //tires burn
+		return
+
+/obj/vehicle/sealed/car/civ/Bump(atom/A)
+	. = ..()
+	if(!isliving(A))
+		return
+	var/mob/living/L = A
+	for(var/i in return_drivers())
+		if(!ishuman(i))
+			continue
+		var/mob/living/carbon/human/boomer = i
+		boomer.say(pick("HOLY SHIT MY PAINT IS RUINED", "I LEASED THIS YOU DICK!", "I WILL END YOU GET OUT OF THE WAY!", "WHAT THE FUCK DO YOU KNOW HOW EXPENSIVE CARS ARE?", "GET OF THE ROAD YOU BRAINDEAD TROGLODYTE", "GET OUT OF THE GODDAMN WAY"), forced="car crash")
```

Comment:
```suggestion
		boomer.say(pick("HOLY SHIT MY PAINT IS RUINED!!", "I LEASED THIS YOU DICK!!", "GET OUT OF THE WAY, I WILL END YOU!!", "WHAT THE FUCK!? DO YOU KNOW HOW EXPENSIVE CARS ARE?!!", "GET OUT OF THE ROAD YOU BRAINDEAD TROGLODYTE!!", "GET OUT OF THE GODDAMN WAY!!"), forced="car crash")
```

Reply:
it's get off the road i typo'd it, get out of the road sounds odd

Reply:
the americans say it

Reply:
wtf

Reply:
i changed it